statsmodels/discrete/discrete_model.py

```diff
@@ -1555,6 +1579,32 @@ def score(self, params):
         else:
             return score
 
+    def score_factor(self, params):
+        if self._transparams:
+            alpha = np.exp(params[-1])
+        else:
+            alpha = params[-1]
+
+        params = params[:-1]
+        p = self.parameterization
+        exog = self.exog
+        y = self.endog[:,None]
+        mu = self.predict(params)[:,None]
+        mu_p = np.power(mu, p)
+        a1 = 1 + alpha * mu_p
+        a2 = mu + alpha * mu_p * y
+        a3 = alpha * p * mu ** (p - 1)
+        a4 = a3 * y
+        dmudb = mu
+
+        dalpha = (mu_p * (y * ((y - 1) / a2 - 2 / a1) + a2 / a1**2))
+        dparams = dmudb * (-a4 / a1 +
+                           a3 * a2 / (a1 ** 2) +
+                           (1 + a4) * ((y - 1) / a2 - 1 / a1) +
+                           1 / mu)
```

Comment:
Take the two terms `-a4 / a1` and `(1 + a4) * ((y - 1) / a2 - 1 / a1)`:

```
(- a4 / a1
 + (1 + a4) * ((y - 1) / a2 - 1 / a1))

= (1 / a1
   - 1 / a1
   - a4 / a1
   + (1 + a4) * ((y - 1) / a2 - 1 / a1))

= (1 / a1
   - (1 + a4) / a1
   + (1 + a4) * ((y - 1) / a2 - 1 / a1))

= (1 / a1
   + (1 + a4) * ((y - 1) / a2 - 2 / a1))
```

The term multiplying `(1+a4)` here also shows up in `dalpha`